src/explore-education-statistics-admin/src/pages/release/datablocks/components/FeaturedTablesTable.tsx

```diff
@@ -0,0 +1,197 @@
+  onSaveOrder: () => Promise<void>;
+}
+
+export default function FeaturedTablesTable({
+  canUpdateRelease,
+  dataBlocks,
+  featuredTables,
+  publicationId,
+  releaseId,
+  onDelete,
+  onReorder,
+  onSaveOrder,
+}: Props) {
+  const [isReordering, toggleIsReordering] = useToggle(false);
+
+  return (
+    <>
+      <div className="dfe-flex dfe-justify-content--space-between">
+        <h3 className="govuk-!-margin-bottom-2">Featured tables</h3>
+        {canUpdateRelease && featuredTables.length > 1 && (
+          <Button
+            className="govuk-!-margin-bottom-2"
+            variant="secondary"
+            onClick={async () => {
+              if (isReordering) {
+                await onSaveOrder();
+              }
+              toggleIsReordering();
+            }}
+          >
+            {isReordering ? 'Save order' : 'Reorder featured tables'}
+          </Button>
+        )}
+      </div>
+
+      <DragDropContext
+        onDragEnd={result => {
+          if (!result.destination) {
+            return;
+          }
+          const reordered = reorder(
+            featuredTables,
+            result.source.index,
+            result.destination.index,
+          ).map((table, index) => ({
+            ...table,
+            order: index,
+          }));
+          onReorder(reordered);
+        }}
+      >
+        <Droppable droppableId="footnotes" isDropDisabled={!isReordering}>
+          {(droppableProvided, droppableSnapshot) => (
+            <table data-testid="featuredTables">
+              <thead>
+                <tr>
+                  <th scope="col" className="govuk-!-width-one-quarter">
+                    Name
+                  </th>
+                  <th scope="col">Has chart</th>
+                  <th scope="col">In content</th>
+                  <th scope="col">Featured table name</th>
+                  <th scope="col">Created date</th>
+                  <th scope="col" className="govuk-table__header--actions">
+                    Actions
+                  </th>
+                </tr>
+              </thead>
+              <DroppableArea
+                droppableProvided={droppableProvided}
+                droppableSnapshot={droppableSnapshot}
+                tag="tbody"
+              >
+                {orderBy(featuredTables, 'order').map(
+                  (featuredTable, index) => (
+                    <DraggableItem
+                      hideDragHandle
+                      id={featuredTable.id}
+                      index={index}
+                      isReordering={isReordering}
+                      key={featuredTable.id}
+                      tag="tr"
+                    >
+                      <FeaturedTablesRow
+                        canUpdateRelease={canUpdateRelease}
+                        dataBlock={dataBlocks.find(
+                          dataBlock =>
+                            dataBlock.id === featuredTable.dataBlockId,
+                        )}
```

Comment:
If there isn't a matching data block, we could end up with a wrapping `DraggableItem` that renders nothing. 

I suspect this could potentially cause weird bugs when reordering so would suggest we reorganise this code a little so that we don't render the `DraggableItem` in this case i.e.

```tsx
{orderBy(featuredTables, 'order').map(
  (featuredTable, index) => {
    const dataBlock = dataBlocks.find(dataBlock => dataBlock.id === featuredTable.dataBlockId);

    return dataBlock ? <DraggableItem>...</DraggableItem> : null;
  }
)}
```

As a side benefit, the `dataBlock` prop would no longer need to be optional on `FeaturedTablesRow`.